central/pruning/pruning.go

```diff
@@ -588,6 +598,18 @@ func (g *garbageCollectorImpl) removeOrphanedNetworkFlows(clusters set.FrozenStr
 	log.Info("[Network Flow pruning] Completed")
 }
 
+func (g *garbageCollectorImpl) removeOrphanedNetworkEntities() {
```

Comment:
Should probably add this to the pruning test.

Reply:
I implemented some pruning testing at https://github.com/stackrox/stackrox/blob/9c15430f634e342859584984c4fb38f59e54ea29/central/networkgraph/entity/datastore/internal/store/postgres/full_store_test.go#L97 .

`removeOrphanedNetoworkEntities()` merely calls in the pruner. I could still test the feature-flag logic; but I must admit that I don't know how to set the flags in the test :sweat_smile: